AsyncDisplayKit/ASMapNode.mm

```diff
@@ -190,6 +183,23 @@ - (void)resetSnapshotter
   _snapshotter = [[MKMapSnapshotter alloc] initWithOptions:_options];
 }
 
+- (void)createInitialOptions
+{
+  _options = [[MKMapSnapshotOptions alloc] init];
+  //Default world-scale view
```

Comment:
Ah, I guess this might prevent _options from being nil below, but...how about putting the nil check in applySnapshotOptions, and if _options is nil, then perform these three lines?  In that case you can actually delete -createInitialOptions as a method, which I don't think is really needed.